source/ip/icmp.md

```
@@ -0,0 +1,247 @@
+  +------------+-----------+-------------+-----------
+  | MACヘッダ  | IPヘッダ  | ICMPヘッダ  | データ...
+  +------------+-----------+-------------+-----------
+```
+
+そして、ICMPヘッダとデータ部分に注目すると、以下のようになっています。
+
+- 1オクテットの『タイプ』
+- 1オクテットの『コード』
+- 2オクテットの『チェックサム』
+- 2オクテットの『識別子』(データ)
+    - データ部分の長さや、種類に応じたデータや元パケットの情報などが適宜渡される
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)